.github/workflows/web_build.yml

```diff
@@ -1,15 +1,20 @@
-###
-name: Automatic web build
+name: "Automatic web build"
 
 on:
   push:
     branches:
-      - master
+      - "master"
+
 jobs:
   build_web_site:
-    runs-on: [ web-builder-22.04 ]
+    runs-on: "ubuntu-latest"
     steps:
-      - name: Build_web
-        run: /scripts/build_web.sh
-      
-
+      - name: "Trigger deployment workflow"
+        run: |
+          gh workflow run deploy-openssl-library-org.yml
```

Comment:
Same question here as in https://github.com/openssl/technical-policies/pull/109/files#r1875984050